proteus/WaveTools.pyx

```diff
@@ -43,26 +43,56 @@ def loadExistingFunction(funcName, validFunctions):
         if func.__name__ == funcName:
             func_ret = func
     if funcName not in funcNames:
-        logEvent("WaveTools.py: Wrong function type (%s) given: Valid wavetypes are %s" %(funcName,funcNames), level=0)
+        logEvent("ERROR! Wavetools.py: Wrong function type (%s) given: Valid wavetypes are %s" %(funcName,funcNames), level=0)
```

Comment:
Is this the approach you use for exception handling?  If so I like it and will use the same format in the future.